R/fit.R

```diff
@@ -641,6 +660,32 @@ return_codes <- function() {
 }
 CmdStanFit$set("public", name = "return_codes", value = return_codes)
 
+#' Return profiling data
+#'
+#' @name fit-method-profiles
+#' @aliases profiles
+#' @description The `$profiles()` method returns a list of data frames with
+#'   profiling data if any profiling data was written to the profile CSV files.
+#'   See [save_profile_files()] to control where the files are saved.
+#'
+#'   Support for profiling Stan programs is available with CmdStan >= 2.26 and
+#'   requires adding profiling statements to the Stan program.
+#'
+#' @return A list of data frames with profiling data if the profiling CSV files
+#'   were created.
+#'
+#' @seealso [`CmdStanMCMC`], [`CmdStanMLE`], [`CmdStanVB`], [`CmdStanGQ`]
+#'
```

Comment:
Can we either add an examples section or link somewhere that has an example of how to do profiling?

Reply:
I will add an example.

Reply:
Ok thanks. Feel free to merge after that.